config/initializers/alexa_ruby_kit_fix.rb

```diff
@@ -0,0 +1,8 @@
+module AlexaRubykit
+
+  # Returns true if all the Alexa request objects are set.
+  def self.valid_alexa?(request_json)
+    !request_json.nil? && !request_json['session'].nil? &&
+        !request_json['request'].nil?
```

Comment:
If you don't 1-line an expression containing multiple boolean operators, then give each condition its own line:
```ruby
request_json.present? &&
  request_json['session'].present? &&
  request_json['request'].present?
```